src/series/Pokemon.yml

```diff
@@ -45,17 +36,20 @@ games:
         platforms: []
         release_year: null
         sub-series: Core
+        comment: Red (international) / Blue (international), Yellow, Gold/Silver, Crystal, Ruby / Sapphire, Emerald, FireRed / LeafGreen
     (I-V) Gens I to V all:
         genres: []
         platforms: []
         release_year: null
         sub-series: Core
+        comment: Red (international) / Blue (international), Yellow, Gold/Silver, Crystal, Ruby / Sapphire, Emerald, FireRed / LeafGreen, Diamond / Pearl, Platinum, HeartGold / SoulSilver, Black and White, Black & White Version 2
     (I-VII) Gens I to VI all:
         genres: []
         platforms: []
         release_year: null
         sub-series: Core
-    (II) Gen II secondary:
+        comment: Red (international) / Blue (international), Yellow, Gold/Silver, Crystal, Ruby / Sapphire, Emerald, FireRed / LeafGreen, Diamond / Pearl, Platinum, HeartGold / SoulSilver, Black and White, Black & White Version 2, X / Y, Omega Ruby / Alpha Sapphire
+    (II) Pokémon Crystal Version:
```

Comment:
the diff is weird to read here, but I renamed `(II) Gen II secondary` to `(II) Pokémon Crystal Version`